### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Shewolf227**
- Use media queries in CSS file to make the desktop version look like the template I selected (Desktop: 768px or wider).
- Use Grid and Flexbox to lay out the elements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,7 +206,7 @@ <h1 class="about-me">About me</h1>
           </ul>
         </div>
         <hr class="line" />
-        <ul class="grid-item">
+        <ul class="grid-item skills-desktop">
           <li class="language-shape skills-heading">
             Language
             <ul class="languages skills-list">
```
Comment (ShadyShawkat) [inline index.html:212]: - [x] Kindly change the bullet points' positioning in the mobile version to match the Figma design.
- [x] Please remove the extra spacing on both sides of this section to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -222,16 +239,12 @@ <h1 class="about-me">About me</h1>
...
-        <p>
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-        </p>
+    <section class="contact-section contact-form">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/mayvbgrw" method="POST">
```
Comment (ShadyShawkat) [inline index.html:247]: - Kindly try to match the Figma design as much as possible, which includes:
   - [x] Adjusting the spacing around each element to match the Figma design.
   - [x] Using the same font style (family, color, size, ...etc) as the Figma design.
   - [x] Adjusting elements' widths (like your form's width) to be more like the Figma design.
   - [x] Increasing the space between the social media links to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -16,71 +16,88 @@
...
-      </p>
+      <div class="headline-div">
+        <h1 class="hey-there">Hey There. I am Sonia</h1>
+        <h2 class="soft-dev">I am a Software Developer</h2>
+        <p class="desc">
+          I can help you build a product, feature or website. Look through some of
+          my work and experience! If you like what you see and have a project you
+          need coded, don’t hestiate to contact me.
+        </p>
+      </div>
```
Comment (ShadyShawkat) [inline index.html:43]: - [x] Kindly increase the top spacing above this section's social links to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -16,71 +16,88 @@
...
+        </li>
+        <li>
+          <img class="item-icon" src="img/social-icons/light-icons/Vector-3.png" alt="Twitter Logo" />
+        </li>
       </ul>
     </section>
     <section class="works">
       <div class="section-title">
         <h1 class="recent-works">My Recent Works</h1>
+        <hr class="recent-works-line">
```
Comment (ShadyShawkat) [inline index.html]: - [x] [OPTIONAL] Kindly try to use only one `h1` tag in the whole web page. This is considered good practice and is more professional. You can use other heading tags depending on the hierarchy of your headings in the web page.
- [x] Kindly increase this horizontal line's length to match the Figma design.
- [x] Please increase the left and right spacing of the works section

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 5]
```diff
@@ -176,7 +193,7 @@ <h2 class="art-title">Profesional Art Printing Data</h2>
     </section>
     <section class="about-me-section">
       <div class="grid-container about">
-        <div class="grid-item about-img">
+        <div class="grid-item grid-item1 about-img">
           <h1 class="about-me">About me</h1>
           <p class="about-desc">
             Hello I’m a software developer! I can help you build a product ,
```
Comment (ShadyShawkat) [inline index.html]: - [x] Kindly adjust the spacing in this section to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html (hunk 1 above)
Comment (ShadyShawkat) [inline index.html:212]: - [x] Kindly try to make this section take the whole width to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 6]
```diff
@@ -222,16 +239,12 @@ <h1 class="about-me">About me</h1>
...
-        </p>
+    <section class="contact-section contact-form">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/mayvbgrw" method="POST">
         <input
           type="text"
           name="name"
```
Comment (ShadyShawkat) [inline index.html:250]: - Kindly try to stick as much as possible to the Figma design, which includes:
   - [x] Adjusting the spacing around this section to match the Figma design.
   - [x] Using the same font style from the Figma design.
   - [x] Adding a horizontal line above the social media icons.
   - [x] Adjusting the width of the "Get in touch" button's width to match the Figma design.

| Your design | Figma design | 
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @SoniaKarungi, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Shewolf227) [reply index.html:212]: Was a tricky one but I think I finally did it! Thanks for all the feedback. Working on it.

File: index.html [hunk 7]
```diff
@@ -176,8 +193,8 @@ <h2 class="art-title">Profesional Art Printing Data</h2>
     </section>
     <section class="about-me-section">
       <div class="grid-container about">
-        <div class="grid-item about-img">
-          <h1 class="about-me">About me</h1>
+        <div class="grid-item grid-item1 about-img">
+          <h2 class="about-me">About me</h2>
```
Comment (gemmen29) [inline index.html:197]: Kindly adjust the spacing in the about section as the previous reviewer asked.
| Your design | Figma design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @SoniaKarungi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
# Status : Approved :tada: 
133696300-b506e476-dc1a-443e-935a-c6f434e788b6


Hi @SoniaKarungi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**